src/autopas/selectors/AutoTuner.h

```diff
@@ -149,7 +154,7 @@ class AutoTuner {
     if (_tuningStrategy->searchSpaceIsTrivial()) {
       return false;
     }
-
+    // TODO This will not return true if forceRetune is called during a tuning phase while sampling something.
```

Comment:
todo?

Reply:
wait this is already fixed via 1f852ca :)